source/_patterns/01-molecules/content-header/content-header.twig

```diff
@@ -4,6 +4,40 @@
 
   {%- block body -%}
 
+    {% if subjects.items is iterable %}
+
+      {%- with subjects only -%}
+
+        {% set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['content-header__subject_list']) }) %}
+
+        <ol {%- include 'atoms-attributes' %}>
```

Comment:
While this matches what eLife has, wondering if we should:

1. Add a visually-hidden title.
2. Change the HTML order so the title comes first, then use flexbox to rearrange.

/cc @davidcmoulton

Reply:
A visually hidden (duplicate) title might incur SEO penalties.

I think we want to keep consistent the HTML source order and the order the page elements are rendered, as divergence may be confusing when using screen reader in conjunction with viewing the screen. Wondering @discodavey knows of an `aria` approach we might use on the list to indicate that the list is described by a subsequent title.

Reply:
Without changing the design or rendering, we could address this by adding an `aria-label` attribute on the `<ol>`. It could be populated in the eLife case with something like `"Subject listings that include this article"`, or some such.

Reply:
Can we park this issue for now? Perhaps open another ticket if we feel strongly about re-ordering it?

The `aria-label` can be set in the `.yaml` for the `<ol>` and `<li>` of the category list.

Reply:
As discussed yesterday with @davidcmoulton that the best way at the moment to just cover the screen reader aspect is to add an aria-label to the ol.

The correct way would be to source order the code so the h1 title is first with the list coming after it but this is something that apparently another conversation.

Reply:
I'm resolving this with the addition of the `aria-label` to the data.